### PR TITLE
CX-16047 Create reusable components for AI Assistant in Momentum UI l…

### PR DESCRIPTION
**Pull request by NotPeterKelly**
…ibrary

## Description
Created various UI components for all possible AI Assistant delegates.

## Screenshots:
image
2024-09-17 15_51_01-Components _ Card Ai - Card Ai ⋅ Storybook — Mozilla Firefox

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by NotNestor** [comment]
Could you add a CardAi storybook file

**Comment by NotPeterKelly** [comment]
> Could you add a CardAi storybook file

So I wasn't sure whether these were still things which worked because when I tried to preview these in the browser I couldn't get anything to render out of the box (for any component!).

**Comment by NotNestor** [comment]
You should update the package.json minor version number

File: web-components/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@momentum-ui/web-components",
-  "version": "2.13.23",
+  "version": "2.13.24",
   "author": "Yana Harris",
```
Comment (NotNestor) [inline web-components/package.json:4]: new component would usually be a minor version change. Increment to 2.14.0

File: web-components/src/components/card-ai/tokens/lm-card-ai-tokens.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable no-undef */
+const colors = require("@momentum-ui/tokens/dist/colors.json");
+
```
Comment (NotNestor) [inline web-components/src/components/card-ai/tokens/lm-card-ai-tokens.js:4]: I think you can remove colors here since you're not using them

**Review by NotNestor — APPROVED** [review]

**Pull request merged**